docs/en/operations/table_engines/custom_partitioning_key.md

```diff
@@ -110,15 +109,12 @@ drwxr-xr-x 2 clickhouse clickhouse 4096 Feb  5 12:09 201902_4_6_1
 drwxr-xr-x 2 clickhouse clickhouse 4096 Feb  1 16:48 detached
 ```
 
-The folders '201901_1_1_0', '201901_1_7_1' and so on, are the directories of the parts. Each part relates to a corresponding partition and contains data just for a certain month (the table in this example has partitioning by month).
+The folders '201901_1_1_0', '201901_1_7_1' and so on are the directories of the parts. Each part relates to a corresponding partition and contains data just for a certain month (the table in this example has partitioning by month).
 
-The 'detached' directory contains parts that were detached from the table using the [DETACH](#alter_detach-partition) query. The corrupted parts are also moved to this directory, instead of being deleted. The server does not use the parts from 'detached' directory. You can add, delete, or modify the data in this directory at any time – the server will not know about this until you make the [ATTACH](../../query_language/alter.md#alter_attach-partition) query.
+Note that on the operating server, you cannot manually change the set of parts or their data on the file system, since the server will not know about it. For non-replicated tables, you can do this when the server is stopped, but we do not recommend it. For replicated tables, the set of parts cannot be changed in any case.
```

Comment:
we do not recommend it -> it is not recommended

Reply:
Let's keep as it is: "we do not recommend".
Microsoft's style guide says that is better to use construction "We recommend" instead of "It is(not) recommended". Besides, this text has been already edited by Emi.

Reply:
ClickHouse is an open-source product, not a service by some specific company. So "we" is kind of misleading.